src/main/resources/assets/wakes/lang/en_us.json

```diff
@@ -24,14 +29,22 @@
   "wake_spawn_rule.wakes.only_wakes": "Only wakes",
   "wake_spawn_rule.wakes.only_splashes": "Only splashes",
   "wake_spawn_rule.wakes.disabled": "Disabled",
+  "description.wakes.spawning_rules": "Defines the wakes and splashes spawning rules.",
   "option.wakes.wakes_in_running_water": "Wakes in running water",
+  "description.wakes.wakes_in_running_water": "Should wakes spawn in running water?",
 
   "option.wakes.wave_speed": "Wave speed",
+  "description.wakes.wave_speed": "The speed which the wakes travel and despawn at.",
   "option.wakes.initial_wave_strength": "Initial wave strength",
+  "description.wakes.initial_wave_strength": "The distance the wakes will travel.",
```

Comment:
Minor thing here: This option might result in wakes traveling farther but that may just be a result of a higher starting value which means it takes longer for the wave to die down. The description should state that it's primarily a strength metric that in turn cause the wakes to travel longer distances.